src/components/site/Accordion.md

```diff
@@ -4,7 +4,7 @@ const collapseItems = [
     id: 1,
     header: "Accordion Item #1",
     subheader: "This is a subheader",
-    content: <h2>Test Heading Level 2</h2>
+    content: <h3>Test Heading Level 3</h3>
```

Comment:
this was adjusted to an h3 per design, as the accordion would never contain an h2, an h2 might be used to describe that instead